modules/statistics/php/charts.class.inc

```diff
@@ -182,8 +182,8 @@ class Charts extends \NDB_Page
             LEFT JOIN session s ON (s.ID=f.SessionID)
             JOIN parameter_type pt USING (ParameterTypeID)
             WHERE pt.Name='acquisition_date'
-            GROUP BY MONTH(pf.Value), YEAR(pf.Value), s.CenterID
-            ORDER BY YEAR(pf.Value), MONTH(pf.Value), s.CenterID",
+            GROUP BY datelabel, s.CenterID
```

Comment:
```
// [...]. Order matters to ensure that the
// labels are calculated in the correct order.
```

This changes the ordering since it goes from ordering by year, month (as integers) to ordering by month-year (as a string, so months are in the wrong order.)

Reply:
The issue is that the value in the select is different then the one in the group by, it seems like MySQL is only happy if u group by the exact value in the select statement. Grouping by one then the other is different then grouping by both as a string but that's what it's selecting for

Any suggestions ?

Reply:
I could select for them separately and have the code concat them...

Reply:
can you add a group by on that without removing the others `GROUP BY MONTH(pf.Value), YEAR(pf.Value), s.CenterID, datelabel`?

Reply:
the problem is selecting on things that aren't in the group by, not grouping by things that aren't in the select.

Reply:
@driusan done